voluncheer/opportunityboard/views/opportunityboard.py

```diff
@@ -0,0 +1,18 @@
+from django.apps import apps
+from django.shortcuts import render
+
+from opportunityboard.models import Opportunity
+
+Organization = apps.get_model("profiles", "Organization")
```

Comment:
I'll believe you if you tell me this is "how it is done in Django" but it surprises me that this isn't just an import statement. Can you point me to the Django documentation for this pattern for my own edification?

Reply:
I don't think this line is necessary because `Organization `is not used in either of the functions. Eden could you please confirm?

Reply:
Confirmed not necessary.